### PR TITLE
A Favour for making Madd Dog Mansion as my interior.

### PR DESCRIPTION
**Pull request by Specifer**
A favor for making Madd Dog Mansion as my house interior.

Thank you,

### CODE REVIEW

**Comment by Specifer** [comment]
@RussellLVP, Can you help me with this. I have no idea...

File: javascript/features/houses/utils/interior_list.js [hunk 1]
```diff
@@ -301,6 +301,33 @@ const INTERIOR_LIST = [
             duration: 2500
         }
     }
```
Comment (RussellLVP) [inline javascript/features/houses/utils/interior_list.js]: There needs to be a comma after this line.

File: javascript/features/houses/utils/interior_list.js [hunk 2]
```diff
@@ -301,6 +301,33 @@ const INTERIOR_LIST = [
...
+            target: [ [ 1265.9929, -793.1546, 1084.0078 ], [ 1250.0294, -780.8049, 1084.1321 ] ],
+            duration: 2500
+        }
+    }
+
+
+
+
+
+
```
Comment (RussellLVP) [inline javascript/features/houses/utils/interior_list.js]: Please remove all the excess lines.

**Review by RussellLVP — COMMENTED** [review]
I'm OK with adding this and changing your interior, but please fix up this PR to not revert your own changes (there currently is no diff), and apply the

File: javascript/features/houses/utils/interior_list.js (hunk 1 above)
Comment (Specifer) [reply javascript/features/houses/utils/interior_list.js]: k

**Comment by Specifer** [comment]
Done!

File: javascript/features/houses/utils/interior_list.js [hunk 3]
```diff
@@ -300,6 +300,27 @@ const INTERIOR_LIST = [
...
+        features: {
+            health: [ 1279.7493, -810.5056, 1085.6328 ],
+            armour: [ 1279.9301, -813.2347, 1085.6328 ],
+            safe: [ 1263.9841, -787.3984, 1091.9063 ]
+        },
+
+        preview: {
+            position: [ [ 1264.8002, -787.5640, 1091.9063 ], [ 1274.2072, -771.9368, 1091.9063 ] ],
+            target: [ [ 1265.9929, -793.1546, 1084.0078 ], [ 1250.0294, -780.8049, 1084.1321 ] ],
```
Comment (RussellLVP) [inline javascript/features/houses/utils/interior_list.js:321]: Out of interest, how did you create those?

Comment (Specifer) [reply javascript/features/houses/utils/interior_list.js:321]: yes, used /save and tracked co-ords. However, IDK if it works! With /int 55

**Review by RussellLVP — APPROVED** [review]
That's probably fine since it's private anyway. Approved.

**Pull request merged**